number_test.go

```diff
@@ -334,6 +336,148 @@ func TestNumber_InDelta(t *testing.T) {
 	}
 }
 
+func TestNumber_InDeltaRelative(t *testing.T) {
+	cases := map[string]struct {
+		number           float64
+		reference        float64
```

Comment:
nit: could you please use "value" instead of "reference"? (or other name). To avoid confusion with AssertionFailure.Reference, which may be part of equality assertion (but not in this case).